src/drivers/xmc.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#if MG_ENABLE_TCPIP && defined(MG_ENABLE_DRIVER_XMC) && MG_ENABLE_DRIVER_XMC
+
+struct mg_tcpip_driver_xmc_data {
+  // MDC clock divider. MDC clock is derived from ETH MAC clock (SYS_CLK / 2)
+  // It must not exceed 2.5MHz
+  //    HCLK range    DIVIDER    mdc_cr VALUE
+  //    -------------------------------------
+  //                                -1  <-- tell driver to guess the value
+  //    60-100 MHz    HCLK/42        0
+  //    100-150 MHz   HCLK/62        1
+  //    20-35 MHz     HCLK/16        2
+  //    35-60 MHz     HCLK/26        3
+  //    150-250 MHz   HCLK/102       4  <-- value for Nucleo-F* on max speed
+  //    250-300 MHz   HCLK/124       5
+  //    110, 111 Reserved
+  int mdc_cr;  // Valid values: -1, 0, 1, 2, 3, 4, 5
```

Comment:
"This ain't no Nucleo", please update that
Are valid values OK ?

Reply:
Yeah, the values are valid (I took the table from Nucleo and only updated the values, forgot the rest..). It is fixed now.